.github/ISSUE_TEMPLATE.md

```diff
@@ -1,20 +1,61 @@
-**Please provide the following information**
+<!--
+###############################################################################
+  WARNING!
+  IGNORING THE FOLLOWING TEMPLATE WILL RESULT IN THE ISSUE BEING CLOSED AS INCOMPLETE
+###############################################################################
+-->
+
+## Checklist
+
+<!--
+################################## IMPORTANT ##################################
+
+As you read and fulfill each of the following listed requirements below,
+put an "x" (without the quotes) between the square brackets.
+
+Ensure that there is no space between the brackets and the "x", like so: [x]
+
+Be honest!
```

Comment:
(:

Reply:
What does this mean?

Reply:
It's a smile because this seems obvious :)

Reply:
But do you think it should be removed?

Reply:
No big deal if you keep it but i'd say yes.

Reply:
Ok. Unless someone else objects, I am inclined to keep it.